### PR TITLE
Fn cli multi arch changes

### PR DESCRIPTION
**Pull request by ssunny3141**
- Link to issue this resolves

- What I did

- How I did it

- How to verify it

- One line description for the changelog

- One moving picture involving robots (not mandatory but encouraged)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,10 +2,12 @@ version: 2
 jobs:
   build:
     machine:
-      image: circleci/classic:201708-01
+      image: ubuntu-2004:202101-01
```
Comment (theatul) [inline .circleci/config.yml:5]: why did we moved to an ubuntu image from circleci?

**Comment by yufeifly** [comment]
您好，您的邮件已收到，谢谢。——熊宇飞

File: .circleci/config.yml (hunk 1 above)
Comment (ssunny3141) [reply .circleci/config.yml:5]: It is no longer working so had to update the image.

**Review by sudhindraaoracle — APPROVED** [review]
Approved based on verification done by Sunny

**Comment by yufeifly** [comment]
您好，您的邮件已收到，谢谢。——熊宇飞

**Pull request merged**